gym_xiangqi/xiangqi_game.py

```diff
@@ -156,6 +168,19 @@ def render(self):
             self.screen.blit(self.cur_selected.select_image,
                              self.cur_selected.get_pygame_coor())
 
+        # update agent_kills and enemy_kills
+        for i in range(len(self.agent_kills)):
+            # keep minimis within the box
+            x = 530 + (i * 35) % 245
```

Comment:
Could these magic numbers be factored into constants? If not, then please explain more in the comments.

Reply:
Since coord (x, y) change based on the current number of kills during the game, I don't this can be directly obtained from constants. However, yes I added more comments to explain them.